.github/workflows/manual-deploy-testnet-l2.yml

```diff
@@ -215,7 +199,7 @@ jobs:
         uses: azure/CLI@v1
         with:
           inlineScript: |
-            az vm run-command invoke -g Testnet -n "${{vars.RESOURCE_STARTING_NAME}}-${{ matrix.host_id }}-${{ GITHUB.RUN_NUMBER }}"  \
+            az vm run-command invoke -g Testnet -n "${{ vars.AZURE_RESOURCE_PREFIX }}-${{ matrix.host_id }}-${{ GITHUB.RUN_NUMBER }}"  \
```

Comment:
nit: `AZURE_RESOURCE_PREFIX` -> `AZURE_RESOURCE_PREFIX_L2`

Reply:
Ah.. you mean that if it's sepolia then everything is `S` ? Independent of what it is.
Yeah fair 👍